packages/blade/src/components/Table/types.ts

```diff
@@ -72,6 +72,11 @@ type TableProps<Item> = {
    * The object should have an id property that is a unique identifier for the row.
    */
   data: TableData<Item>;
+  /**
+   * Selection mode determines how the table rows can be selected.
+   * @default 'row'
+   **/
+  multiSelectionMode?: 'checkbox' | 'row';
```

Comment:
Should we call this `multiSelectTrigger?: 'checkbox' | 'row'`

Reply:
Sounds good to me.